keyboards/crkbd/keymaps/mcrown/keymap.c

```diff
@@ -70,6 +70,12 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   ),
 };
 
+#ifdef BOTH_KEYBOARDS_OLED
```

Comment:
I'm not sure why this is needed. 

If you enable `SPLIT_OLED_ENABLE`, it will sync the on-off state from the master to the secondary side, so that it will turn on both sides when there is activity.